tests/test_resolver.py

```diff
@@ -106,20 +108,54 @@ def test_resolver_private_count(self):
 
     def test_resolver_write(self):
         # write out csv and read in and check
-        exp = 'GenusA speciesA,|kingdomA|phylumA|orderA|familyA|genusA|speciesA,\
-test_source,1,1.0,|kingdom|phylum|order|family|genus|species,GenusA \
-speciesA,GenusA speciesA,|51|41|31|21|11|1,3|0|0,1,1,test_number1'
+        exp_header = 'query_name,classification_path,data_source_title,\
```

Comment:
Similarly to the issue with the taxTrees, the test_resolver_write methods gave inconsistent ordering of results lines when run in Python3. Rather than explicitly sorting the output, I instead rewrote the test to compare all the output lines and the header instead of just the first output line.